Bugfix v3.1.2 (#566)
admin-shell-io/aas-specs-api: documentation/IDTA-01002-3/modules/ROOT/pages/includes/constraints.adoc

:aasa001: pass:q[[underline]#Constraint AASa-001:# The value of the xref:http-rest-api/http-rest-api.adoc#pagination[cursor] query parameter must not be empty. If the client does not know the cursor value, it must omit the whole query parameter in the request.]

> :aasa002: pass:q[[underline]#Constraint AASa-002#: The base64url-encoded identifier of the link: https://industrialdigitaltwin.io/aas-specifications/IDTA-01001/V3.2/spec-metamodel/common.html#has-semantics-attributes[semanticId] shall have a length of maximum 3072 characters.]

Author: ```suggestion
:aasa002: pass:q[[underline]#Constraint AASa-002#: The base64url-encoded identifier of the link: https://industrialdigitaltwin.io/aas-specifications/IDTA-01001/V3.1/spec-metamodel/common.html#has-semantics-attributes[semanticId] shall have a length of maximum 3072 characters.]
```

> :aasa003: pass:q[[underline]#Constraint AASa-003#: A service implementing the link:https://industrialdigitaltwin.io/aas-specifications/IDTA-01002/V3.2/http-rest-api/service-specifications-and-profiles.html#submodel-repositoryservice-specification-SSP-003[SubmodelRepositoryServiceSpecification/SSP-003] must not accept or provide any link:https://industrialdigitaltwin.io/aas-specifications/IDTA-01001/V3.2/spec-metamodel/core.html#submodel-attributes[Submodel]  with the attribute link:https://industrialdigitaltwin.io/aas-specifications/IDTA-01001/V3.2/spec-metamodel/common.html#ModellingKind[kind="Instance"].]

Author: ```suggestion
:aasa003: pass:q[[underline]#Constraint AASa-003#: A service implementing the link:https://industrialdigitaltwin.io/aas-specifications/IDTA-01002/V3.1/http-rest-api/service-specifications-and-profiles.html#submodel-repositoryservice-specification-SSP-003[SubmodelRepositoryServiceSpecification/SSP-003] must not accept or provide any link:https://industrialdigitaltwin.io/aas-specifications/IDTA-01001/V3.1/spec-metamodel/core.html#submodel-attributes[Submodel]  with the attribute link:https://industrialdigitaltwin.io/aas-specifications/IDTA-01001/V3.1/spec-metamodel/common.html#ModellingKind[kind="Instance"].]
```

> :aasa004: pass:q[[underline]#Constraint AASa-004#: A service implementing the link:https://industrialdigitaltwin.io/aas-specifications/IDTA-01002/V3.2/http-rest-api/service-specifications-and-profiles.html#_submodel_repository_template_read_profile[SubmodelRepositoryServiceSpecification/SSP-004] must not accept or provide any _Submodel_ with the attribute link:https://industrialdigitaltwin.io/aas-specifications/IDTA-01001/V3.2/spec-metamodel/common.html#ModellingKind[kind="Instance"].]

Author: ```suggestion
:aasa004: pass:q[[underline]#Constraint AASa-004#: A service implementing the link:https://industrialdigitaltwin.io/aas-specifications/IDTA-01002/V3.1/http-rest-api/service-specifications-and-profiles.html#_submodel_repository_template_read_profile[SubmodelRepositoryServiceSpecification/SSP-004] must not accept or provide any link:https://industrialdigitaltwin.io/aas-specifications/IDTA-01001/V3.1/spec-metamodel/core.html#submodel-attributes[Submodel] with the attribute link:https://industrialdigitaltwin.io/aas-specifications/IDTA-01001/V3.1/spec-metamodel/common.html#ModellingKind[kind="Instance"].]
```

> :aasa005: pass:q[[underline]#Constraint AASa-005:# Only the _SubmodelElements_ root delaration can be followed with link:https://industrialdigitaltwin.io/aas-specifications/IDTA-01001/V3.2/mappings/mappings.html#_format_path_idshortpath_serialization_in_json[IdShortPaths].]

Author: ```suggestion
:aasa005: pass:q[[underline]#Constraint AASa-005:# Only the _SubmodelElements_ root delaration can be followed with link:https://industrialdigitaltwin.io/aas-specifications/IDTA-01001/V3.1/mappings/mappings.html#_format_path_idshortpath_serialization_in_json[IdShortPaths].]
```
